README.md

```diff
@@ -1,3 +1,7 @@
+[![CharmHub Badge](https://charmhub.io/nginx-ingress-integrator/badge.svg)](https://charmhub.io/nginx-ingress-integrator)
+[![Publish to edge](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/test_and_publish_charm.yaml/badge.svg)](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/test_and_publish_charm.yaml)
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
+
```

Comment:
Can we add one additional badge for the release libraries action?

Reply:
I didn't find a workflow for that to add a badge.

For example, this one has because it has a specific workflow for releasing libraries.

The workflow:
https://github.com/canonical/prometheus-k8s-operator/actions/workflows/release-libs.yaml/

The badge:
[![Release Libraries](https://github.com/canonical/prometheus-k8s-operator/actions/workflows/release-libs.yaml/badge.svg)](https://github.com/canonical/prometheus-k8s-operator/actions/workflows/release-libs.yaml)

Reply:
Oh, I see. The action would have to be in its own workflow, is that it?

Reply:
Understood. thx